plugins/linus_quotes/linus_quotes.rb

```diff
@@ -0,0 +1,33 @@
+#  -*- coding: utf-8 -*-
+#  linus_quotes.rb
+#  Author: Winston Weinert
+#  ------------------------
+#  A Cinch plugin that provides Linus Torvalds quotes for yossarian-bot.
+#  linus_quotes.txt generated from Wikiquote's article
+#  ------------------------
+#  This code is licensed by Winston Weinert under the MIT License.
+#  http://opensource.org/licenses/MIT
+
+require_relative '../yossarian_plugin'
+
+class LinusQuotes < YossarianPlugin
+	include Cinch::Plugin
+	use_blacklist
+
+	QUOTES_FILE = File.expand_path(File.join(File.dirname(__FILE__), 'linus_quotes.txt'))
+	QUOTES = File.readlines(QUOTES_FILE)
+
+	def usage
+		'!linus [nick] - Fetch a random Linus Torvalds quote.'
```

Comment:
Small nit: Say "Aliases: !torvalds" or something similar at the end of the usage string.